src/workflows/modals/CreateWorkflowModal.tsx

```diff
@@ -77,15 +77,15 @@ const createWorkflowModalConstraints = {
   },
 };
 
-interface NamespaceNameSectionProps {
+interface CollectionAndNameSectionProps {
```

Comment:
Would it make sense to also modify namespace to collection inside the interface as well? Also what does name refers to here, the collection name or the workflow name?

Reply:
I had intentionally decided to rename only user facing and component names to `collection` because we had made a decision in previous renaming efforts to only change high level component names and user-facing places to new terminology. It also wouldn't be trivial to update all the instances of `namespace` to `collection` everywhere in code. And even if we do change the variable names we would still need to refer these as namespace (and snapshot) whenever we are using the Ajax calls so that it can be consistent with backend terminology. 

Having said that, I do understand that there might be some confusion when someone else comes and looks at the code. So I am open to the discussion on renaming what else what be good for developer readability.

Reply:
Gotcha, in that case maybe a little comment in the code explaining that a namespace is the collection, and name is the workflow name?

Reply:
Yes I can do that. I will also update the component name to be `CollectionAndWfNameSection` so it is more clear.